packages/documentation/src/stories/components/icon/icon.stories.tsx

```diff
@@ -1,65 +1,77 @@
-import React from 'react';
-import { Meta, Args, Story } from '@storybook/react';
+import { Args, Story } from '@storybook/react';
 import { BADGE } from '@geometricpanda/storybook-addon-badges';
 import docsPage from './icon.docs.mdx';
 
 import { forEach } from '../../../utils/react';
 
 export default {
   title: 'Components/Icon',
+  component: 'post-icon',
   parameters: {
     docs: {
       page: docsPage,
     },
     badges: [BADGE.NEEDS_REVISION],
   },
   args: {
-    'name': '1022',
-    'base': '',
-    'flip-h': false,
-    'flip-v': false,
-    'scale': 1,
-    'rotate': 0,
-    'animation': 'null',
+    name: '1022',
+    base: '',
+    flipH: false,
+    flipV: false,
+    scale: null,
+    rotate: null,
+    animation: 'null',
   },
   argTypes: {
-    'name': {
+    name: {
       name: 'Name',
-      control: {
-        type: 'text',
+      table: {
+        category: 'General',
       },
     },
-    'base': {
+    base: {
       name: 'Base',
-      control: {
-        type: 'text',
+      table: {
+        category: 'General',
       },
     },
-    'flip-h': {
+    flipH: {
       name: 'Flip Horizontally',
-      control: {
-        type: 'boolean',
+      table: {
+        category: 'Transformation',
       },
     },
-    'flip-v': {
+    flipV: {
       name: 'Flip Vertically',
```

Comment:
Here the argument table describes the API of the web component, so I'd rather show the property with its real name and not override it. What do you think?